operators/src/util/input.rs

```diff
@@ -0,0 +1,262 @@
+use crate::error;
+use failure::_core::convert::TryFrom;
+use serde::de::Visitor;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+/// A type that allows use inputs to be either strings or numbers
+#[derive(Debug, Clone, PartialEq)]
+pub enum StringOrNumber {
```

Comment:
use https://crates.io/crates/either crate to save code

Reply:
Would have been an option indeed, but we have three variants, `either` provides two. And we don't need the dependency now, since I tailor-fitted it to our data types :smile:.

Reply:
Either<String,Either<Number, Decimal>> ;) Anyway I think the implementation could be improved see my other comment

Reply:
Voting for keeping the own implementation now.